### PR TITLE
Give our RadioButton a retouch

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

It now shares the same transition time and "bounce" as the
CheckBox, which makes them resemble each other far more.

Its label now aligns itself better as well.

# Result

**Before**

https://user-images.githubusercontent.com/69514187/232163632-6b5f5248-a62a-4a29-897d-a0b6ef509847.mov

**After**

https://user-images.githubusercontent.com/69514187/232163455-a8f69e3d-6613-492c-baff-459f4293b989.mov

https://user-images.githubusercontent.com/69514187/232163444-4a2bdfb1-d0a4-4fe2-ba7d-0ad65b8a0b1d.mov

# Testing

- [x] I have thoroughly tested my changes.

Tested radio buttons in and outside of forms.

### CODE REVIEW

File: app/components/Form/RadioButton.css [hunk 1]
```diff
@@ -1,90 +1,95 @@
-.box {
```
Comment (ivarnakken) [inline app/components/Form/RadioButton.css:1]: The entire file is rewritten, so there's no need to compare changes.

**Review by ollfkaih — APPROVED** [review]
Looks much more like the checkboxes, I like it :))
Cleaner code also in Joblisting, which is always nice

File: app/routes/joblistings/components/JoblistingRightNav.tsx [hunk 2]
```diff
@@ -150,64 +150,44 @@ const JoblistingsRightNav = (props: Props) => {
...
-          htmlFor="deadline"
-          style={{
-            cursor: 'pointer',
+        <RadioButton
+          name="sort"
+          id="deadline"
+          label="Frist"
+          checked={order.deadline}
+          onChange={() => {
+            props.history.push({
```
Comment (LudvigHz) [inline app/routes/joblistings/components/JoblistingRightNav.tsx:161]: Would be nice if you could replace these with `replace` as well :smile:

**Review by LudvigHz — APPROVED** [review]
Awesome!

File: app/routes/joblistings/components/JoblistingRightNav.tsx (hunk 2 above)
Comment (ivarnakken) [reply app/routes/joblistings/components/JoblistingRightNav.tsx:161]: Out of scope here. Write it up as a todo and we can go over all of these

**Pull request merged**